src/ralph/operations/api.py

```diff
@@ -13,9 +13,17 @@ class Meta:
         depth = 1
 
 
+class OperationStatusSerializer(RalphAPISerializer):
+
+    class Meta:
+        model = OperationStatus
+        depth = 1
+
+
 class OperationSerializer(RalphAPISerializer):
     type = OperationTypeSerializer()
     assignee = SimpleRalphUserSerializer()
+    status = OperationStatusSerializer()
```

Comment:
Maybe SlugRelatedField (http://www.django-rest-framework.org/api-guide/relations/#slugrelatedfield) will be better here? You can define this field like that:
```
status = SlugRelatedField(
    many=False,
    read_only=False,
    slug_field='name',
    queryset= OperationStatus.objects.all()
)
```

Should work :)